examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -114,7 +117,7 @@ angular.module('cloudberry.common', [])
       return filter;
     }
 
-    function byGeoRequest(parameters) {
+    function byGeoRequest(parameters, geoIds) {
```

Comment:
if `getFilter` doesn't use `geoIds`, neither does `byGeoRequest`.

Reply:
Lines 291 and 322 use it differently.